spec/ddtrace_integration_spec.rb

```diff
@@ -6,14 +6,7 @@
     subject(:shutdown) { Datadog.shutdown! }
 
     let(:start_tracer) do
-      # TODO: We manually unset the tracer instance here as
-      # we don't have a mechanism to deeply reset the
-      # `Datadog.configuration` object.
-
-      # Ensure we terminate the existing tracer instance, if any
-      Datadog.configuration.tracer.instance.shutdown! if Datadog.configuration.tracer.instance
-      Datadog.configure { |c| c.tracer.instance = nil }
-
+      Datadog.configure {}
       Datadog.tracer.trace('test.op') {}
```

Comment:
Minor: Is the extra `configure` call really needed here? E.g. wouldn't `Datadog.tracer` trigger any needed initialization anyway?

Reply:
I believe you are correct, trying it.